core/templates/components/state-directives/outcome-editor/outcome-destination-editor.directive.html

```diff
@@ -5,7 +5,7 @@
   </div>
   <div class="form-group oppia-form-group">
     <span>
-      <select class="form-control oppia-form-control" ng-model="$ctrl.outcome.dest" ng-options="choice.id as choice.text for choice in $ctrl.destChoices" ng-change="$ctrl.onDestSelectorChange()">
+      <select id="oppia-form-control" class="form-control" ng-model="$ctrl.outcome.dest" ng-options="choice.id as choice.text for choice in $ctrl.destChoices" ng-change="$ctrl.onDestSelectorChange()">
```

Comment:
Why are you using id? We try not to do this since it is hard to guarantee a single id per page.

Reply:
Removed id